### PR TITLE
turbojpeg library update for rocAL

### PR DESCRIPTION
**Pull request by rrawther**
Removes private branch for turbojpeg.
Upgraded turbojpeg to public branch 3.0.1.
Need to re-run setup script for the changes to take effect

### CODE REVIEW

File: rocAL-setup.py [hunk 1]
```diff
@@ -335,7 +335,7 @@
 
     # turbo-JPEG - https://github.com/rrawther/libjpeg-turbo.git -- 2.0.6.2
     os.system(
-        '(cd '+deps_dir+'; git clone -b 2.0.6.2 https://github.com/rrawther/libjpeg-turbo.git )')
+        '(cd '+deps_dir+'; git clone -b 3.0.1 https://github.com/libjpeg-turbo.git )')
```
Comment (SundarRajan98) [inline rocAL-setup.py:338]: Broken repo link - change to https://github.com/libjpeg-turbo/libjpeg-turbo.git

**Comment by SundarRajan98** [comment]
@rrawther Adding the changes from your [fork](https://github.com/rrawther/libjpeg-turbo/tree/2.0.6.2) to [upstream](https://github.com/libjpeg-turbo/libjpeg-turbo/tree/3.0.1) libjpeg-turbo library works and rocAL trainings converge with the expected accuracy in rocm 6.0 docker. But the rocAL trainings doesn't converge with this PR changes so I compared the changes from your fork to this PR changes - found some changes in tjDecompress2_partial and tjDecompress2_partial_scale functions. Removed the [changes](https://github.com/SundarRajan98/rocAL/commit/f46a680c862bc8063c2ac8c6645b31d709547499) and triggered rocAL resnet50 trainings which converges with the expected performance. 
Note: A libjpeg error stating "Unsupported color conversion request" is being printed for each epoch with this PR changes

**Comment by rrawther** [comment]
@kiritigowda : Can we merge this PR. @SundarRajan98 has tested it and we can remove the tjpeg dependency from my fork

**Comment by rrawther** [comment]
@kiritigowda : For this PR, the turbojpeg library is different version which I have updated in the setup script. But I think the CI is using the old version to build and hence it is failing

**Pull request merged**